app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/forms.scala

```diff
@@ -533,3 +533,13 @@ object SelectTopicsFromSubscriptionsForm {
     )(SelectTopicsFromSubscriptionsForm.apply)(SelectTopicsFromSubscriptionsForm.unapply)
   )
 }
+
+final case class NoApplicationsChoiceForm(choice: Option[String])
```

Comment:
We're putting these in companion objects of controllers nowadays

Reply:
will do